index.bs

```diff
@@ -1020,9 +1020,12 @@ Expectations and Discipline</h4>
 	including suspending or removing for cause
 	a participant in any group (including the [=AB=] and [=TAG=])
 	if serious and/or repeated violations,
-	such as failure to meet the requirements of this process
+	such as failure to meet the requirements on individual behavior of
+	(a) this process
 	(where repeated attempts to address these violations have not resolved the situation)
```

Comment:
I don't think we should repeat "repeated" here, it's in the beginning. and if the violation is sufficiently serious, as the sentence starts, once may be enough.

Reply:
Good point. :) I think we needed the parenthetical in the old wording, but not since we restructured this section.

Reply:
Agreed. removed the parenthetical.